src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.cs

```diff
@@ -341,6 +341,13 @@ public override void VisitAlias(IAliasSymbol symbol)
             }
         }
 
+        internal override void VisitPreprocessing(IPreprocessingSymbol symbol)
```

Comment:
This change should be backed by a test under Compilers

Reply:
Other symbols like range variable and function pointers don't seem to be tested in `SymbolVisitorTests`. Is it necessary to include specified tests?

Reply:
This comment refers to symbol display changes.